HumanResourceMachine/Controllers/HumanController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Net;
+using HumanResourceMachine.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace HumanResourceMachine.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HumanController : ControllerBase
+    {
+        // Temporary database
+        private static List<Human> people = new List<Human>()
+        {
+            new Human
+            {
+                Id = 1,
+                Name = "Ilya",
+                Surname = "Safonov",
+                Patronymic = "Alexandrovich"
+            },
+            new Human
+            {
+                Id = 2,
+                Name = "SomeName",
+                Surname = "SomeSurname",
+                Patronymic = "SomePatronymic"
+            }
+        };
+
+        [HttpGet]
+        public IActionResult GetAllPeople()
+        {
+            return Ok(people);
+        }
+
+        [HttpGet("{id}")]
+        public IActionResult GetHumanById(int id)
```

Comment:
Just Get instead of GetHumanById, add [FromRoute] for id,  return Human

Reply:
Done.